.pipelines/ci.yml

```diff
@@ -618,7 +618,10 @@ stages:
         python ./tools/ios/build_xcframework.py \
           --output_dir $(Build.BinariesDirectory)/xcframework_out \
           --platform_arch iphonesimulator x86_64 \
-          --config Release
+          --config RelWithDebInfo \
+          --ios_deployment_target 13.0 \
```

Comment:
iOS 13.0 required for unit tests due to use of std::filesystem. if we are testing with iOS 13, should we just make iOS 13 the official minimum version?